manifests/istio-control/istio-discovery/templates/configmap-jwks.yaml

```diff
@@ -1,11 +0,0 @@
-{{- if .Values.pilot.jwksResolverExtraRootCA }}
```

Comment:
Please don't just remove this. This is afaik currently the only way to inject trust for users who host their JWKS on a server with a cert signed by an internal CA